WordPress/Classes/ViewRelated/Reader/Filter/FilterTableData.swift

```diff
@@ -0,0 +1,50 @@
+struct TableDataItem {
+    let topic: ReaderAbstractTopic
+    let configure: (UITableViewCell) -> Void
+}
+
+class FilterTableViewDataSource: NSObject, UITableViewDataSource {
+
+    let data: [TableDataItem]
+    private let reuseIdentifier: String
+
+    init(data: [TableDataItem], reuseIdentifier: String) {
+        self.data = data
+        self.reuseIdentifier = reuseIdentifier
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return data.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let item = data[indexPath.row]
+        let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath)
+
+        item.configure(cell)
+
+        return cell
+    }
+}
+
+class SiteTableViewCell: UITableViewCell, GhostableView {
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
+        detailTextLabel?.textColor = UIColor.systemGray
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func ghostAnimationWillStart() {
+        contentView.subviews.forEach { view in
+            view.isGhostableDisabled = true
+        }
+        textLabel?.text = "TEST LABEL Text Label"
```

Comment:
Should we remove these test titles from this PR?

Reply:
Yep, this was a bad attempt at sizing these labels but without having to manipulate the default frames. I've switched to repeating characters with a given width; I don't really love this, but it seems to work alright.

https://github.com/wordpress-mobile/WordPress-iOS/blob/de1b872f75f11397fa04d4285d12a393335bbbcd/WordPress/Classes/ViewRelated/Reader/Filter/FilterTableData.swift#L50